### ISSUE TITLE
Git tagging fails on release publish

### ISSUE THREAD

**Issue by cowboyd** [thefrontside/actions#12]
The release action run here https://github.com/microstates/microstates.js/commit/5ba88cd2614c340bb1f62fd60184505a47608739/checks?check_suite_id=257270796 failed because of what appears to be the `-a` option being passed to `git tag` which creates an "annotated" tag and requires input on what the annotation is. Without the `-m` flag it will open up an editor, but since this is a CI env it fails.

I think this is unnecessary and can be removed.

https://github.com/thefrontside/actions/blob/master/synchronize-with-npm/entrypoint.sh#L28-L29

Also, the git command to push the tags does not need to push any branches, so it should be

```
$ git push "https://$GITHUB_ACTOR:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git" --tags
```

**Comment by minkimcello** [comment]
@cowboyd Yeah that was my mistake. I slipped in the -a tag at the last second thinking it wouldn't hurt to have extra information with the tag but documentation says message is necessary with the -a tag. I've taken that off on my forked `actions` repository and will submit a PR along with the new synchronize-npm-tags action tomorrow.

And for the branch, I just tested pushing tags without it and it worked fine. I've never worked with tags before so I wasn't sure but I took that out too.

Another thing I overlooked was I hard-coded 'microstates' for checking previous published npm versions. I'll make that dynamic as well.

My bad. I'll try to be more tidy in the future.

**Comment by cowboyd** [comment]
No worries! Honestly, I would have been shocked if it had worked the first time.

**Comment by minkimcello** [comment]
Addressed in [PR#13](https://github.com/thefrontside/actions/pull/13).

**Comment by cowboyd** [comment]
Good question. I imagine that it's repo /org dependent. We'll see!
